native/swift/Tests/wordpress-api/Support/CocoaPods.swift

```diff
@@ -0,0 +1,15 @@
+#if COCOAPODS
+
+import Foundation
+
+private class BundleFinder {}
+
+extension Bundle {
+    static var module: Bundle {
+        let url = Bundle(for: BundleFinder.self).url(forResource: "Resources", withExtension: "bundle")!
+        try? Bundle(for: BundleFinder.self).bundleURL.absoluteString.write(toFile: "/Users/tonyli/Projects/wordpress-rs/file.path", atomically: true, encoding: .utf8)
```

Comment:
> /Users/tonyli/

Is that ^ intentional? If it can be any path, maybe it should be non-user specific to carry its intention?

Reply:
Good catch! It's a debugging code that I forgot to delete.